learn/protocol/sync-protocol/pico-macro-sync.md

```diff
@@ -0,0 +1,15 @@
+# Pico Macro Sync
+
+Pico Macro Sync is a macro sync mechanism used by pico nodes to quickly determine the canonical chain state by querying peers and comparing their responses. It does not rely on ZKPs verification but instead uses a lightweight, peer-based trust approach. After syncing with the macro chain, the syncing peers follows with the [Block Live Sync](block-live-sync.md).
+
+The Pico Macro Sync operates on a per-peer basis. For each connected peer, the syncing node performs the following steps:
+
+1. Request epoch IDs from the peer.
+2. Request the latest election or checkpoint blocks from the peer, if available.
+3. The node compares the received state with its own. If the information matches, the peer is considered consistent. If the peer is missing expected data or provides conflicting information, it is treated as inconsistent and may be excluded from the sync process.
```

Comment:
if the peer provides conflicting information, we fallback to light macro sync. Since we are not using proofs, we don't have any way to identify if a peer is being malicious, it is behind or we have inconsistent data. So if we find any discrepancy we fallback to light macro sync